model/networks.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, input_dim, cnum, use_cuda=True, device_ids=None):
         self.use_cuda = use_cuda
         self.device_ids = device_ids
 
-        self.conv1 = gen_conv(input_dim, cnum, 5, 1, 2)
+        self.conv1 = gen_conv(input_dim + 2, cnum, 5, 1, 2)
```

Comment:
Sorry, I am a little confused. Why we use `input_dim + 2` here?

Reply:
I see. You have changed the `input_dim` in `config.yaml`. And the `+2` here means concatenating the ones and mask. Good job!